packages/core/tsconfig.json

```diff
@@ -14,6 +14,12 @@
     "target": "ES2015",
     "resolveJsonModule": true,
   },
+  "ts-node": {
+    "files": true,
```

Comment:
just curious, Is this necessary? I didn't see the usage inside core pacakge

Reply:
@himself65 on the documentation is recommend to use ts-node to run the examples and do some tests, I wasn't able to use without this

Reply:
I mean, this config is inside `package/core/tsconfig.json`, if you want run examples, should it put inside the top level?

Reply:
on the top level it didn't work, but will investigate why, thanks for pointing it!